recursion/intro-to-recursion.md

```diff
@@ -0,0 +1,135 @@
+1. Remove the line segment that is the base of the triangle from step two
+
+The [Fibonacci sequence](https://en.wikipedia.org/wiki/Fibonacci_number) is a sequence of numbers where each next number is defined in terms of previous numbers in the sequence. As the sequence grows, the ratios between consecutive terms approaches the golden ratio **𝛗**, which is often considered a marker of natural beauty.
+
+## Examples from Nature
+
+- Romanesco broccoli, where each bud is made up of smaller buds
+- Sourdough bread that was made from leftover sourdough
+- Branching behavior in certain plants and algae, where a branch branches off another branch
+
+We can observe many fractal patterns—patterns which have internal similarity—in nature. They can often be described recursively.
+
+Many claim to find the golden ratio (related to the recursive Fibonacci sequence) hidden within numerous natural structures.
+
+- Sunflower seed heads
+- Pinecone seeds
+- Flower petals
+
+## Examples from Art
+
+- Matryoshka dolls
+  - Wooden dolls that contain other matryoshka dolls within itself
+- Droste effect
+  - An effect where an image contains the image itself
+- Infinity mirrors
+
+## Check for Understanding
+
+<!-- Question 1 -->
+<!-- prettier-ignore-start -->
+### !challenge
+* type: checkbox
+* id: sE7nhe
+* title: Thinking Recursively
+##### !question
+
+Which of the following are examples of recursion?
+
+##### !end-question
+##### !options
+
+* A triangle which contains another triangle
+* A Sierpiński triangle made up of other Sierpiński triangles
+* Calculating the fibonacci sequence, in which the next number in the sequence is the sum of the last two numbers in the sequence
+* A row of similar jars and containers on a shelf
+* A painted self-portrait
+
+##### !end-options
+##### !answer
+
+A Sierpiński triangle made up of other Sierpiński triangles is defined by referring to itself.
+
+<br />
+
+Calculating the fibonacci sequence follows a recursive algorithm. Its solution depends on the solutions of smaller versions of the same problem.
+
+<br />
+
+Putting a triangle in another triangle doesn't necessarily make it recursive. If that inner triangle also had an inner triangle, and _that_ inner triangle also had an inner triangle, and so on, then it could be recursive.
+
+<br />
+
+Similar jars aren't defined in terms of each other, so would not be recursive. They just happen to be similar.
+
+<br />
+
+A painted self portrait where the artist was painting a self portrait of themselves painting a self portrait, and so on, could be recursive, as in the Droste effect. But a typical self portrait would not be recursive.
+
+##### !end-answer
+##### !explanation
+
+A Sierpiński triangle made up of other Sierpiński triangles is defined by referring to itself.
+
+Calculating the fibonacci sequence follows a recursive algorithm. Its solution depends on the solutions of smaller versions of the same problem.
```

Comment:
same merge issue?
```suggestion
A Sierpiński triangle made up of other Sierpiński triangles is defined by referring to itself.

<br />

Calculating the fibonacci sequence follows a recursive algorithm. Its solution depends on the solutions of smaller versions of the same problem.

<br />

Putting a triangle in another triangle doesn't necessarily make it recursive. If that inner triangle also had an inner triangle, and _that_ inner triangle also had an inner triangle, and so on, then it could be recursive.

<br />

Similar jars aren't defined in terms of each other, so would not be recursive. They just happen to be similar.

<br />

A painted self portrait where the artist was painting a self portrait of themselves painting a self portrait, and so on, could be recursive, as in the Droste effect. But a typical self portrait would not be recursive.
```